tests/output/pjs/test_utils.js

```diff
@@ -8,6 +8,10 @@
  *          or a boolean indicating that errors are expected.
  *  test: A callback function to run with all the results
  *  skip: skip the test
+ *  only: only run this test
+ *  setup: custom setup callback, takes a single argument: output
+ *  teardown: custom teardown callback, takes a single argument: output
+ *  wait: time to wait after first code run, this allows "draw" to be called
```

Comment:
Added some new options that we helpful/necessary to write the tests to verify that this bug has been fixed.